Add Fleet audit logging memory leak to 8.8.0-8.8.1 known issues (#160566)
elastic/kibana: docs/CHANGELOG.asciidoc

Review the following information about the {kib} 8.8.1 release.

[float]
[[known-issues-8.8.1]]
=== Known issues

// tag::known-issue-159807[]
[discrete]

> .Memory leak in Fleet audit logging.
> [%collapsible]
> ====
> *Details* +
> Fleet introduced audit logging for various CRUD (create, read, update, and delete) operations in 8.8.0.
> 
> While audit logging is not enabled by default, we have identified an off-heap memory leak in the implementation of Fleet audit logging that can result in poor Kibana performance and in some cases, Kibana instances getting terminated by the OS kernel's oom-killer. This memory leak can occur even when Kibana audit logging is not explicitly enabled (regardless of whether `xpack.security.audit.enabled` is set in the kibana.yml).
> 
> *Impact* +
> The 8.8.2 release includes in {kibana-pull}159807[a fix] for this problem. Customers using Fleet integrations
> and Kibana audit logging in 8.8.0 and 8.8.1 should upgrade to 8.8.2 or above to obtain the fix.
> ====

Reviewer: ```suggestion
.Memory leak in {fleet} audit logging.
[%collapsible]
====
*Details* +
{fleet} introduced audit logging for various CRUD (create, read, update, and delete) operations in version 8.8.0.

While audit logging is not enabled by default, we have identified an off-heap memory leak in the implementation of {fleet} audit logging that can result in poor {kib} performance, and in some cases {kib} instances being terminated by the OS kernel's oom-killer. This memory leak can occur even when {kib} audit logging is not explicitly enabled (regardless of whether `xpack.security.audit.enabled` is set in the `kibana.yml` settings file).

*Impact* +
The version 8.8.2 release includes in {kibana-pull}159807[a fix] for this problem. If you are using {fleet} integrations
and {kib} audit logging in version 8.8.0 or 8.8.1, you should upgrade to 8.8.2 or above to obtain the fix.
====
```

====
// end::known-issue-158940[]

// tag::known-issue-159807[]
[discrete]

> .Memory leak in Fleet audit logging.
> [%collapsible]
> ====
> *Details* +
> Fleet introduced audit logging for various CRUD (create, read, update, and delete) operations in 8.8.0.
> 
> This audit logging is not enabled by default. Users must set `xpack.security.audit.enabled: true` in the `kibana.yml` to enable it.
> 
> We have identified an off-heap memory leak in Fleet audit logging that can result in Kibana instances getting terminated by the operating system kernel's `oom-killer`.
> 
> *Impact* +
> The 8.8.2 release includes in {kibana-pull}159807[a fix] for this problem. Customers using Fleet integrations
> and Kibana audit logging in 8.8.0 and 8.8.1 should upgrade to 8.8.2 or above to obtain the fix.
> ====

Reviewer: ```suggestion
.Memory leak in {fleet} audit logging.
[%collapsible]
====
*Details* +
{fleet} introduced audit logging for various CRUD (create, read, update, and delete) operations in version 8.8.0.
While audit logging is not enabled by default, we have identified an off-heap memory leak in the implementation of {fleet} audit logging that can result in poor {kib} performance, and in some cases {kib} instances being terminated by the OS kernel's oom-killer. This memory leak can occur even when {kib} audit logging is not explicitly enabled (regardless of whether `xpack.security.audit.enabled` is set in the `kibana.yml` settings file).
*Impact* +
The version 8.8.2 release includes in {kibana-pull}159807[a fix] for this problem. If you are using {fleet} integrations
and {kib} audit logging in version 8.8.0 or 8.8.1, you should upgrade to 8.8.2 or above to obtain the fix.
====
```
